GameEngine/Players/GreatestDistanceSingleCardPlayer.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace GameEngine.Players
+{
+    public class GreatestDistanceSingleCardPlayer : Player
+    {
+        public override Play FormulatePlay(GameBoard board)
+        {
+            Play greatestSingleDistancePlay = null;
+            int greatestDistance = 0;
+            foreach (var owlPosition in board.Owls.ListOfPositions) {
+                foreach (var card in Hand)
+                {
+                    var play = new Play(card, owlPosition);
+                    var newPosition = board.FindDestinationPosition(play);
+                    var distance = newPosition - owlPosition;
```

Comment:
❓ Do we even need to do the subtraction? I.e. would `greatestNewPosition` accomplish the same thing?

Reply:
We chatted about this. :-)